BUILDING.md

```diff
@@ -35,7 +35,7 @@ distribution in `/usr`.
 ```
 $ meson build
 $ ninja -C build # or `meson compile -C build`
-$ sudo ninja -C build install # or `meson install -C build`
+$ ninja -C build install # or `meson install -C build`
```

Comment:
Why unnecessary? I think it is necessary to have sudo privileges to install in /usr/local. What system are you using?

Reply:
Actually, could you add `sudo` also to the instructions in README.md? On most systems `sudo` will be needed to install to /usr/local

Reply:
on Mac OS it's not required but I'm fine changing both to sudo

Reply:
i do agree with ret2libc. the only one that should not have sudo is when you install the binary locally.

Reply:
ok let me change everywhere then